### PR TITLE
[ICP-5938, ICP-5937, ICP-5936, ICP-5933] Orvibo Dual Gang Switch - Address several issues with child/parent interaction

### PR DESCRIPTION
**Pull request by varzac**
You should review by commit as I update the format in the zigbee-multi-switch to use tab indenting, but it makes the diff gross.

### CODE REVIEW

File: devicetypes/smartthings/child-switch-health.src/child-switch-health.groovy [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+			}
+		}
+
+		main "switch"
+		details(["switch"])
+	}
+}
+
+def installed() {
+	sendEvent(name: "checkInterval", value: hcIntervalMinutes * 60, displayed: false, data: [protocol: "zigbee", hubHardwareId: device.hub.hardwareID])
```
Comment (tpmanley) [inline devicetypes/smartthings/child-switch-health.src/child-switch-health.groovy]: Where is `hcIntervalMinutes` set?

Comment (varzac) [reply devicetypes/smartthings/child-switch-health.src/child-switch-health.groovy]: Nowhere, I just got distracted and forgot to finish it.  I also haven't tested this yet.

Comment (tpmanley) [reply devicetypes/smartthings/child-switch-health.src/child-switch-health.groovy]: Looks like a copy paste issue. I think it makes sense to have a default checkInterval set here but the real value should be set by the parent to whatever it wants.

File: devicetypes/smartthings/child-switch-health.src/child-switch-health.groovy [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+void off() {
+	parent.childOff(device.deviceNetworkId)
+}
+
+def ping() {
+	// Intentionally left blank as parent should handle this
+}
+
+def uninstalled() {
+	parent.delete()
```
Comment (tpmanley) [inline devicetypes/smartthings/child-switch-health.src/child-switch-health.groovy:53]: Can you add the Jira for the delete issue to the title?

File: devicetypes/smartthings/child-switch-health.src/child-switch-health.groovy [hunk 3]
```diff
@@ -41,9 +45,6 @@ void off() {
 }
 
 def ping() {
-	parent.ping()
+	// Intentionally left blank as parent should handle this
```
Comment (tpmanley) [inline devicetypes/smartthings/child-switch-health.src/child-switch-health.groovy:49]: Does anything bad happen if we return null here? I don't recall if that's valid or if we need to do something else to make it a noop.

File: devicetypes/smartthings/zigbee-multi-switch.src/zigbee-multi-switch.groovy [hunk 4]
```diff
@@ -156,8 +156,9 @@ def configureHealthCheck() {
 		runEvery5Minutes("healthPoll")
 		// Device-Watch allows 2 check-in misses from device
 		sendEvent(name: "checkInterval", value: hcIntervalMinutes * 60, displayed: false, data: [protocol: "zigbee", hubHardwareId: device.hub.hardwareID])
+		def childEvent = createEvent([name: "checkInterval", value: hcIntervalMinutes * 60, displayed: false, data: [protocol: "zigbee", hubHardwareId: device.hub.hardwareID]])
```
Comment (tpmanley) [inline devicetypes/smartthings/zigbee-multi-switch.src/zigbee-multi-switch.groovy]: Could create the eventMap once to decrease the potential for them getting set differently

File: devicetypes/smartthings/child-switch-health.src/child-switch-health.groovy [hunk 5]
```diff
@@ -32,6 +32,10 @@ metadata {
 	}
 }
 
+def intalled() {
+	sendEvent(name: "checkInterval", value: 30 * 60, displayed: false, data: [protocol: "zigbee", hubHardwareId: device.hub.hardwareID])
```
Comment (tpmanley) [inline devicetypes/smartthings/child-switch-health.src/child-switch-health.groovy]: Add a comment indicating this is set to a default value but should be overridden by the parent.

**Review by tpmanley — DISMISSED** [review]

**Comment by tpmanley** [comment]
Needs testing before merge

**Review by rappleg — DISMISSED** [review]

**Review by tpmanley — DISMISSED** [review]

File: devicetypes/smartthings/zigbee-multi-switch.src/zigbee-multi-switch.groovy [hunk 6]
```diff
@@ -17,126 +17,122 @@
...
-                attributeState "turningOff", label:'${name}', action:"switch.on", icon:"st.switches.light.off", backgroundColor:"#ffffff", nextState:"turningOn"
-            }
-        }
-        standardTile("refresh", "device.refresh", inactiveLabel: false, decoration: "flat", width: 2, height: 2) {
-            state "default", label:"", action:"refresh.refresh", icon:"st.secondary.refresh"
-        }
-        main "switch"
-        details(["switch", "refresh"])
-    }
+	definition(name: "ZigBee Multi Switch", namespace: "smartthings", author: "SmartThings", ocfDeviceType: "oic.d.switch", vid: "generic-switch") {
```
Comment (tpmanley) [inline devicetypes/smartthings/zigbee-multi-switch.src/zigbee-multi-switch.groovy]: @greens , we need the `mnmn` here too right?

Comment (greens) [reply devicetypes/smartthings/zigbee-multi-switch.src/zigbee-multi-switch.groovy]: As far as I know. Weird that it wasn't there before.

`mnmn: "SmartThings"`

**Comment by varzac** [comment]
updated

**Review by tpmanley — APPROVED** [review]

**Pull request merged**